index.js

```diff
@@ -1,35 +1,65 @@
-/*!
- * forwarded
- * Copyright(c) 2014 Douglas Christopher Wilson
- * MIT Licensed
- */
-
-/**
- * Module exports.
- */
+'use strict'
 
-module.exports = forwarded
+var Processor = require('./lib/processor')
+var schemas = require('./lib/schemas')
 
 /**
  * Get all addresses in the request, using the `X-Forwarded-For` header.
  *
- * @param {Object} req
+ * @param {http.IncomingMessage} req
  * @api public
  */
 
-function forwarded(req) {
+module.exports = function forwarded (req, options) {
+  options = options || {}
+
+  var opts = {
+    // default to only common + standard
+    // array order matters here
+    schemas: options.schemas || [
```

Comment:
> to clarify are you suggesting to keep RFC 7239 as the default option?

Sure that works. Specifically, I'm suggesting that the default option should be _only one schema_; which one it is really doesn't matter as much, because people will change it.

> another possible option is to not offer a default

Well, that is a possibility, yes, but I'm undecided on that (though if you did it, you'd need to move the `schemas` out of the `options` argument, since it's no longer optional).